frontend/src/components/ScratchList.tsx

```diff
@@ -118,19 +126,25 @@ export function ScratchItemNoOwner({ scratch }: { scratch: api.TerseScratch }) {
     const compilerName = compilersTranslation.t(scratch.compiler)
     const serverPresets = api.usePlatforms()[scratch.platform].presets
     const presetName = serverPresets.find(p => p.id === scratch.preset)?.name
+    const preset = serverPresets.find(p => p.id === scratch.preset)
```

Comment:
I believe if you put this preset line above the presetName one, presetName can just be `preset?.name` rather than doing this find() twice